### PR TITLE
[Simple] Fix a broken link in doc/table_of_contents

### PR DESCRIPTION
**Pull request by yuntai**

### CODE REVIEW

File: doc/table_of_contents.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - [blocks_and_headers](chain/blocks_and_headers.md) - How Grin tracks blocks and headers on the chain
 - [contractideas](contractideas.md) - Ideas on how to implement contracts
 - [dandelion/dandelion](dandelion/dandelion.md) - About transaction propagation and cut-through. Stemming and fluffing!
-- [dandelion/simulating](dandelion/simulating.md) - Dandelium simulation - aggregating transaction without lock_height Stemming and fluffing!
+- [dandelion/simulation](dandelion/simulation.md) - Dandelium simulation - aggregating transaction without lock_height Stemming and fluffing!
```
Comment (quentinlesceller) [inline doc/table_of_contents.md]: Could you fix the following typo:
It's not ```Dandelium simulation``` it's ```Dandelion simulation```

**Comment by quentinlesceller** [comment]
Hi @yuntai, by any chance could you also fix the typo mentioned above ?

File: doc/table_of_contents.md (hunk 1 above)
Comment (yuntai) [reply doc/table_of_contents.md]: done!

**Comment by quentinlesceller** [comment]
Thanks merged!

**Pull request merged**